test/conformance/tests/sriov_operator.go

```diff
@@ -880,9 +880,9 @@ var _ = Describe("operator", func() {
 			})
 		})
 
-		Context("unhealthyVfs", func() {
-			// 25834
-			It(" Should not be able to create pod successfully if there are only unhealthy vfs", func() {
+		Context("PF shutdown", func() {
+			// 29398
+			It("Should be able to create pods successfully if PF is down.Pods are able to communicate with each other", func() {
```

Comment:
nit: space before "Pods"

nit: "with each other" -> "with each other on the same node"